### PR TITLE
Merge changelogs

### PR DESCRIPTION
**Pull request by kamil-stripe**

### CODE REVIEW

File: WIPv73Changelog.md [hunk 1]
```diff
@@ -501,7 +606,26 @@ Default API version changed to "2022-07-28".
...
   | Plan | `Plan.AggregateUsage` | `string` | `PlanAggregateUsage` | Use enum that was already defined but unused |
-  | Plan | `Plan.TiersMode` | `string` | `PlanTiersMode` | Use enum that was already defined but unused | <!-- ---- R, S -->
+  | Plan | `Plan.TiersMode` | `string` | `PlanTiersMode` | Use enum that was already defined but unused |
+  | Refund | `Refund.SourceTransferReversal` | `*Reversal` | `*TransferReversal` | Rename `Reversal` to TransferRev`ersal for consistency with other Stripe client libraries |
+  | Refund | `Refund.TransferReversal` | `*Reversal` | `*TransferReversal` | Rename `Reversal` to TransferRev`ersal for consistency with other Stripe client libraries |
+  | SetupIntent | `SetupIntentPaymentMethodOptionsACSSDebit.Currency` | `string` | `*SetupIntentPaymentMethodOptionsACSSDebitCurrency` | `SetupIntentPaymentMethodOptionsACSSDebitCurrency` is an alias to `string` type |
+  | Source | `Source.Receiver` | `*ReceiverFlow` | `*SourceReceiver` | TODO |
+  | Source | `Source.Redirect` | `*RedirectFlow` | `*SourceRedirect` | TODO |
```
Comment (yejia-stripe) [inline WIPv73Changelog.md:614]: @kamil-stripe I think these two are just renaming things to be consistent with our naming conventions

File: WIPv73Changelog.md [hunk 2]
```diff
@@ -501,7 +606,26 @@ Default API version changed to "2022-07-28".
   | Person | `PersonParams.AddressKana` | `*AccountAddressParams` | `*PersonAddressKanaParams` | `AccountAddressParams` has been split into separate `PersonAddressKanaParams` and `PersonAddressKanjiParams` |
   | Person | `PersonParams.AddressKanji` | `*AccountAddressParams` | `*PersonAddressKanjiParams` | `AccountAddressParams` has been split into separate `PersonAddressKanaParams` and `PersonAddressKanjiParams` |
   | Plan | `Plan.AggregateUsage` | `string` | `PlanAggregateUsage` | Use enum that was already defined but unused |
-  | Plan | `Plan.TiersMode` | `string` | `PlanTiersMode` | Use enum that was already defined but unused | <!-- ---- R, S -->
+  | Plan | `Plan.TiersMode` | `string` | `PlanTiersMode` | Use enum that was already defined but unused |
+  | Refund | `Refund.SourceTransferReversal` | `*Reversal` | `*TransferReversal` | Rename `Reversal` to TransferRev`ersal for consistency with other Stripe client libraries |
+  | Refund | `Refund.TransferReversal` | `*Reversal` | `*TransferReversal` | Rename `Reversal` to TransferRev`ersal for consistency with other Stripe client libraries |
```
Comment (yejia-stripe) [inline WIPv73Changelog.md:611]: Misplaced backticks

File: WIPv73Changelog.md [hunk 3]
```diff
@@ -501,7 +606,26 @@ Default API version changed to "2022-07-28".
...
+  | SetupIntent | `SetupIntentPaymentMethodOptionsACSSDebit.Currency` | `string` | `*SetupIntentPaymentMethodOptionsACSSDebitCurrency` | `SetupIntentPaymentMethodOptionsACSSDebitCurrency` is an alias to `string` type |
+  | Source | `Source.Receiver` | `*ReceiverFlow` | `*SourceReceiver` | TODO |
+  | Source | `Source.Redirect` | `*RedirectFlow` | `*SourceRedirect` | TODO |
+  | Subscription | `SubscriptionListParams.Created` | `int64` | `*int64` | `Created` is nullable |
+  | Subscription | `SubscriptionListParams.Customer` | `string` | `*string` | `Customer` is nullable |
+  | Subscription | `SubscriptionListParams.Plan` | `string` | `*string` | `Plan` is nullable |
+  | Subscription | `SubscriptionListParams.Price` | `string` | `*string` | `Price` is nullable |
+  | Subscription | `SubscriptionListParams.Status` | `string` | `*string` | `Status` is nullable |
```
Comment (yejia-stripe) [inline WIPv73Changelog.md:619]: Sorry I was wrong about these. The rationale is that Params fields should be pointers

File: WIPv73Changelog.md [hunk 4]
```diff
@@ -501,7 +606,26 @@ Default API version changed to "2022-07-28".
...
+  | Subscription | `Subscription.PauseCollection` | `SubscriptionPauseCollection` | `*SubscriptionPauseCollection` | `PauseCollection` is nullable |
+  | Subscription | `Subscription.PendingInvoiceItemInterval` | `SubscriptionPendingInvoiceItemInterval` | `*SubscriptionPendingInvoiceItemInterval` | `PendingInvoiceItemInterval` is nullable |
+  | SubscriptionItem | `SubscriptionItem.BillingThresholds` | `SubscriptionItemBillingThresholds` | `*SubscriptionItemBillingThresholds` | `BillingThresholds` is nullable |
+  | SubscriptionSchedule | `SubscriptionScheduleListParams.CanceledAt` | `int64` | `*int64` | `CanceledAt` is nullable |
```
Comment (yejia-stripe) [inline WIPv73Changelog.md:623]: here too

File: WIPv73Changelog.md [hunk 5]
```diff
@@ -10,7 +10,17 @@ Default API version changed to "2022-07-28".
 - Add `CheckoutSessionShippingCostTax` and `CheckoutSessionShippingCost` classes
 - Add `IssuingCardCancellationReasonDesignRejected` constant to `IssuingCardCancellationReason`.
 - Add `Validate` field to `Customer` resource.
-- Add `Validate` field to `PaymentSourceParams`. <!-- ---- R, S -->
+- Add `Validate` field to `PaymentSourceParams`.
+- Add `SetupAttemptPaymentMethodDetailsCardThreeDSecureResultExempted` constant in `SetupAttemptPaymentMethodDetailsCardThreeDSecureResult`.
+- Add `SKUPackageDimensionsParams` and `SKUPackageDimensions`.
```
Comment (yejia-stripe) [inline WIPv73Changelog.md:15]: Think this should be moved to Changed

File: WIPv73Changelog.md [hunk 6]
```diff
@@ -10,7 +10,17 @@ Default API version changed to "2022-07-28".
...
+- Add `SourceACHCreditTransfer`, `SourceACHDebit`, `SourceACSSDebit`, `SourceAlipay`, `SourceAUBECSDebit`, `SourceBancontact`, `SourceCard`, `SourceCardPresent`, `SourceEPS`, `SourceGiropay`, `SourceIDEAL`,  `SourceKlarna`, `SourceMultibanco`, `SourceP24`, `SourceRedirect`, `SourceSEPACreditTransfer`, `SourceSEPADebit`, `SourceSofort`, `SourceThreeDSecure`, and `SourceWechat`. Additionally, matching fields in `Source` struct - `ACHCreditTransfer`, `ACHDebit`, `ACSSDebit`, `Alipay`, `AUBECSDebit`, `Bancontact`, `Card`, `CardPresent`, `EPS`, `Giropay`, `IDEAL`, `Klarna`, `Multibanco`, `P24`, `SEPACreditTransfer`, `SEPADebit`, `Sofort`, `ThreeDSecure`, and `Wechat`
+- Add `SourceTransactionACHCreditTransfer`, `SourceTransactionCHFCreditTransfer`, `SourceTransactionGBPCreditTransfer`, `SourceTransactionPaperCheck`, and `SourceTransactionSEPACreditTransfer` and matching `SourceTransaction.ACHCreditTransfer`, `SourceTransaction.CHFCreditTransfer`, `SourceTransaction.GBPCreditTransfer`, `SourceTransaction.PaperCheck`, and `SourceTransaction.SEPACreditTransfer`.
+- Add `SubscriptionDeleteDiscountParams`.
```
Comment (yejia-stripe) [inline WIPv73Changelog.md:18]: This should be covered by

> Move `DelSub` from `discount/client.go` to `subscription/client.go` and rename to `DeleteDiscount`

in Changed

File: WIPv73Changelog.md (hunk 5 above)
Comment (kamil-stripe) [reply WIPv73Changelog.md:15]: which line?

**Pull request merged**